portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -23,10 +26,19 @@
 /** Servlet that returns some example content. TODO: modify this file to handle comments data */
 @WebServlet("/data")
 public class DataServlet extends HttpServlet {
+    
+  private List<String> names;
+ 
+  @Override
+  public void init() {
+    names = new ArrayList<String>(Arrays.asList("Sam", "Hadley", "Jerry"));
+  }
 
   @Override
   public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
+    String name = names.get((int) (Math.random() * names.size()));
     response.setContentType("text/html;");
-    response.getWriter().println("<h1>Hello Sam Sandoval!</h1>");
+    response.getWriter().println("Hello " + name + "!");
```

Comment:
did you want to keep the h1 tags?

Reply:
No, I believe that is being removed, its red on their